python/src/UnionEvent_doc.i.in

```diff
@@ -10,11 +10,12 @@ An occurrence of one single event :math:`E_i` yields the occurrence of the syste
 Parameters
 ----------
 coll : sequence of :class:`~openturns.RandomVector`
-    Collection of events
+    Collection of events, evaluated from left to right.
+    Ordering the most probable events first can help to avoid evaluating the last ones.
```

Comment:
```suggestion
    Ordering the most probable events first can help to avoid evaluating the last ones.
    In case the relative probabilities of the events are unknown,
    they can be ranked quickly with (for example) a FORM analysis.
```